libs/ui-components/src/components/Device/DevicesPage/DevicesPage.tsx

```diff
@@ -30,20 +30,28 @@ import {
 } from '../../Status/utils';
 import EnrollmentRequestList from './EnrollmentRequestList';
 import { FilterStatusMap } from './types';
+import PageWithPermissions from '../../common/PageWithPermissions';
+import { RESOURCE, VERB } from '../../../types/rbac';
+import { useAccessReview } from '../../../hooks/useAccessReview';
 
 type DeviceEmptyStateProps = {
   onAddDevice: VoidFunction;
 };
 
 const DeviceEmptyState: React.FC<DeviceEmptyStateProps> = ({ onAddDevice }) => {
   const { t } = useTranslation();
+  const [canCreateFleet] = useAccessReview(RESOURCE.FLEET, VERB.CREATE);
   return (
     <ResourceListEmptyState icon={MicrochipIcon} titleText={t('No devices here!')}>
       <EmptyStateBody>
-        <Trans t={t}>
-          You can add devices and label them to match fleets, or your can{' '}
-          <Link to={ROUTE.FLEET_CREATE}>start with a fleet</Link> and add devices into it.
-        </Trans>
+        {canCreateFleet ? (
+          <Trans t={t}>
+            You can add devices and label them to match fleets, or your can{' '}
```

Comment:
There was a typo here
```suggestion
            You can add devices and label them to match fleets, or you can{' '}
```

Reply:
fixed